src/modules/Dashboard/features/Applet/Activities/Activities.tsx

```diff
@@ -40,11 +41,6 @@ export const Activities = () => {
     };
   }, [appletId, getActivities]);
 
-  const activities = useMemo(
-    () => (activitiesData?.result ?? []).map((activity) => formatRow(activity)),
-    [activitiesData, formatRow],
-  );
-
   useEffect(() => {
```

Comment:
I noticed this and am wondering if this is an unnecessary `useEffect`. If so, I'll probably remove it. @farmerpaul what do you think?